mysql-test/suite/rocksdb_rpl/t/multiclient_2pc.test

```diff
@@ -3,23 +3,30 @@
 --source include/have_debug.inc
 --source include/have_debug_sync.inc
 --source include/big_test.inc
+# The test involves a crash which does not seem to be handled well with
```

Comment:
This sounds suspicious, I have never seen such hang under Valgrind (only memory allocations at the time crash time reported as leaks), but I am not seeing what could be causing this here. 

You could also add a --source include/not_crashrep.inc here (as documentation that this is a crash-testing testcase; the include file is for Mac and mostly a no-op)